ako-gateway-api/lib/lib.go

```diff
@@ -66,7 +66,7 @@ func GetChildName(parentNs, parentName, routeNs, routeName, matchName string) st
 	if matchName != "" {
 		name = fmt.Sprintf("%s-%s", name, utils.Stringify(utils.Hash(matchName)))
 	}
-	return lib.Encode(name, lib.EVHVS)
+	return lib.EncodeWithPrefix(name, lib.EVHVS)
```

Comment:
@abhishekbsingh Will this impact wcp gw upgrade? Will there be any stale resources and traffic impact?

Reply:
Gateway is introduced in 9.1 only, so no upgrade scenario to handle.